src/test/java/com/endava/webapp/controller/EmployeeControllerTest.java

```diff
@@ -0,0 +1,95 @@
+package com.endava.webapp.controller;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.http.MediaType;
+
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.put;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+class EmployeeControllerTest extends BaseIT {
+
+    @Test
+    void shouldReturnOk_whenGettingAllEmployees() throws Exception {
+        mockMvc.perform(get("/employees?offset=2&limit=3"))
```

Comment:
get() method provides you ability to specify string with placeholders and only then put actual variables.
`get("/employees?offset={offset}&limit={limit}", offset, limit)`

It could be useful if you wanna write parameterized tests. Same for method with PathVariables.

P.S. sorry if I'm mistaken with syntax, but there is should be available such feature at least for PathVars in other tests.

Reply:
updated